variables.tf

```diff
@@ -303,3 +303,9 @@ variable "materialize_instances" {
   }))
   default = []
 }
+
+variable "install_metrics_server" {
+  description = "Whether to install the metrics-server for the Materialize Console"
+  type        = bool
+  default     = true
+}
```

Comment:
Found the correct PR.  Same apologies -- a little out of scope, but am noticing that:

- the `terraform.tfvars.example`'s `materialize_instances` have database_password  and are not commented out. (As an FYI -- in the docs, I'm updating the docs now to use a separate tfvars file for the instances, terraform plan -var-file=terraform.tfvars -var-file=mz_instances.tfvars -out my-plan.tfplan`, so that we don't have to worry about commenting and uncommenting.)

- the install_materialize_operator = false in that file (not sure if we want that to be true)

https://github.com/MaterializeInc/terraform-aws-materialize/blob/main/terraform.tfvars.example#L25C1-L43C2